cvat-ui/src/components/tasks-page/task-item.tsx

```diff
@@ -121,10 +122,21 @@ class TaskItemComponent extends React.PureComponent<TaskItemProps & RouteCompone
 
         return (
             <Col span={10} className='cvat-task-item-description'>
-                <Text strong type='secondary' className='cvat-item-task-id'>{`#${id}: `}</Text>
-                <Text strong className='cvat-item-task-name'>
-                    {name}
-                </Text>
+                {taskInstance.name.length > 70 ? (
```

Comment:
Perhaps we do not need that condition now. Is not it? 
We may use the same scheme with antd ellipse.

Reply:
> Perhaps we do not need that condition now. Is not it? We may use the same scheme with antd ellipse.

Yes we dont need it but by using ellipse the name will be in two lines I have applied the changes you should look at it.

![Uploading Screenshot from 2024-03-13 15-29-12.png…]()